OpenMensa_Parser/OpenMensa_Parser/XmlWriter.cs

```diff
@@ -1,15 +1,107 @@
 using System;
+using System.Text;
 using System.Xml;
+using System.Globalization;
 
 namespace OpenMensa_Parser
 {
-    public class XmlWriter
+    static class XmlWriter
     {
-        public string XmlFilePath {get; private set;}
+        public static string fileName = "OpenmenMensaXml.xml";
+        public static string parserVersion = "1.01-1";
+        public static string feedInformation = "http://openmensa.org/open-mensa-v2";
+        public static string schemaInstance = "http://www.w3.org/2001/XMLSchema-instance";
+        public static string schemaLocation = "http://openmensa.org/open-mensa-v2.xsd";
 
-        public XmlWriter(string filePath)
+        public static int categoryCounter = 0;
+        public static int dishCounter = 0;
+        public static int priceCounter = 0;
+
+        public static string[] roleNames = new string[] {"student", "employee", "other", "pupil"};
+        public static char[] removedCharacters = new char[] {'€', ' '};
+
+        public static void WriteXmlFile()
+        {
+            XmlTextWriter xmlWriter = new XmlTextWriter(fileName, System.Text.Encoding.UTF8);
+            xmlWriter.Formatting = Formatting.Indented;
+
+            WriteOpenMensaStandardInformation(xmlWriter);
+            WriteMenuInformation(xmlWriter);
+        }
+
+        public static void WriteOpenMensaStandardInformation(XmlTextWriter xmlWriter)
+        {            
+            //Writing process:
+            xmlWriter.WriteStartDocument();                 //writes: <?xml version="1.0" encoding="utf-8"?>
+            xmlWriter.WriteStartElement("openmensa");
+            xmlWriter.WriteAttributeString("version", "2.1");
```

Comment:
- should also be a variable version (e.g. might change to 2.2 by the OpenMensa organization)